lib/EventRider.js

```diff
@@ -26,6 +26,8 @@ module.exports = class GoogleAnalyticsEventRider {
 
     if (voxaEvent.request.locale) {
       this.visitor.set('ul', voxaEvent.request.locale.toLowerCase());
+      const countryCode = voxaEvent.request.locale.split('-')[0];
+      this.visitor.set('geoid', countryCode.toUpperCase());
```

Comment:
is this correct? seems like country code for a locale `en-US` would be recognized as `en` when ga  expects `US` ?

@wuelcas

Reply:
Oh man, you're so right. I'll fix this. It worked for me since I tested this with "nl-NL", that's why I didn't notice the bug, thanks.

Reply:
please add a small test so it doesn't happen again